test/models/localization_files_test.rb

```diff
@@ -154,7 +154,7 @@ def test_api_error_translations
         tags << "api_#{Regexp.last_match(1).underscore.tr("/", "_")}".to_sym
       end
     end
-    Dir.glob("#{::Rails.root}/app/classes/api/parsers/*.rb").each do |file|
+    Dir.glob("#{::Rails.root}/app/classes/api2/parsers/*.rb").each do |file|
```

Comment:
See above question.

Reply:
I'd in favor of testing api2 only, because i'm hoping we will see fit to delete API very soon and it's less busywork. But I can add it now, merge, and add the delete to the "Sunset the API" branch if you like.

Reply:
Oh also - I believe they use the same localization files, so testing one should be enough.